Assets/Scripts/SS3D/Systems/Health/BodyParts/HeadBodyPart.cs

```diff
@@ -81,14 +76,26 @@ protected override void BeforeDestroyingBodyPart()
             GetComponentInParent<Human>()?.DeactivateComponents();
         }
 
+        protected override void InflictDamage(BodyLayer layer, DamageTypeQuantity damageTypeQuantity)
```

Comment:
here we have custom logic only when the head is severed, instead of overriding a method that has no direct relation to severing logic, can you maybe use an event or one of the abstract method ?